hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/interfaces/Handler.java

```diff
@@ -223,4 +225,9 @@ public void setClusterID(String clusterID) {
     this.clusterId = clusterID;
   }
 
+  public abstract ContainerCommandResponseProto streamDataReadOnly(
```

Comment:
I would suggest change the function name from streamDataReadOnly to readBlock().

Reply:
Done